### PR TITLE
Tidy up docstrings

### PR DESCRIPTION
**Pull request by jkauffman1**

### CODE REVIEW

File: green_cli/btc.py [hunk 1]
```diff
@@ -159,21 +167,25 @@ def getsubaccounts(session):
 @print_result
 @with_gdk_resolve
 def getsubaccount(session, pointer):
+    """SHow details of specific subaccount."""
```
Comment (JamieDriver) [inline green_cli/btc.py:170]: `Show details...`    (caps)

File: green_cli/btc.py [hunk 2]
```diff
@@ -309,7 +321,10 @@ def getunspentoutputs(session, details):
 @print_result
 @with_gdk_resolve
 def setunspentoutputsstatus(session, details):
-    """Set unspent outputs status. Status format is <txid>:<vout>:[default|frozen]"""
+    """Set unspent outputs status.
+
+    Status format is <txid>:<vout>:[default|frozen].
+    """
```
Comment (JamieDriver) [inline green_cli/btc.py:327]: Not sure about the trailing period when it's a format/command like this (ie. where punctuation characters may be part of the format/literal).

**Comment by JamieDriver** [comment]
utack 6adeca0, assuming two small nits above.

File: green_cli/btc.py (hunk 1 above)
Comment (jkauffman1) [reply green_cli/btc.py]: fixed, thanks

File: green_cli/btc.py (hunk 2 above)
Comment (jkauffman1) [reply green_cli/btc.py]: Yes, good point, removed

**Pull request merged**